docs/api/hooks.md

```diff
@@ -401,7 +383,7 @@ export function useActions(actions, deps) {
       return actions.map(a => bindActionCreators(a, dispatch))
     }
     return bindActionCreators(actions, dispatch)
-  }, deps)
+  }, deps ? [dispatch, ...deps] : deps)
```

Comment:
This is a bugfix for an issue I noticed with my original `useActions` implementation.